engines/scumm/he/sound_he.cpp

```diff
@@ -780,7 +796,11 @@ void SoundHE::playHESound(int soundID, int heOffset, int heChannel, int heFlags,
 	}
 }
 
-void SoundHE::tryLoadSoundOverride(int soundID, Audio::RewindableAudioStream **stream) {
+void SoundHE::tryLoadSoundOverride(int soundID, Audio::RewindableAudioStream **stream, int *duration) {
```

Comment:
It would be nice to return the resulting stream, instead of passing it by reference, i.e.:

    Audio::RewindableAudioStream *SoundHE::tryLoadSoundOverride(int soundID, int *duration) {